verilog/tools/ls/vscode/README.md

```diff
@@ -1,18 +1,20 @@
-# VS Code Extension
+# Verible Language Server Extension
 
-## Installation
-Be sure your system as a recent version of both [nodejs](https://nodejs.org/)
-and [npm](https://www.npmjs.com/).
-From the directory of this Readme, run `npm install`.
+## Features
+The language server provides a couple of features from the [Verible SystemVerilog productivity suite](https://github.com/chipsalliance/verible) right in the editor.
 
-After [installing the verible tools](../../../README.md#installation),
-run `npm run vsix` to generate the extension file `verible.vsix`.
+ * **Linting**: Checks your code against a number of
+   [lint rules](https://chipsalliance.github.io/verible/lint.html) and provides
+   'wiggly lines' with diagnostic output and even offers auto-fixes when available.
+ * **Formatting**: Offers Format Document/Selection according to the Verible
+   formatting style. The 'look' can be configured if needed.
+ * **Outline**: shows the high-level structure of your modules and functions in the
```

Comment:
capitalize Shows (I missed this the first round)

Reply:
Done.